src/include/kernel/task/resource/dropbear_res.h

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * @file
+ * @brief Declaration of structure of addreses for dropbear session structure
+ * @date 24.07.2023
+ */
+
+#ifndef DROPBEAR_RES_H_
+#define DROPBEAR_RES_H_
+
+#include <kernel/task.h>
+
+typedef unsigned int addr_t;
```

Comment:
Maybe it's better to use uintptr_t for this purpose? I mean 64 bits platform can have sizeof(void *) == 8 but sizeof(unsiged int) == 4. But I 'm not sure

Reply:
sure thing, you're absolutely right